projects/aca-playwright-shared/src/page-objects/components/adf-info-drawer.component.ts

```diff
@@ -36,4 +36,32 @@ export class AdfInfoDrawerComponent extends BaseComponent {
   public getIdField = (labelText: string) => this.getChild('[data-automation-id="library-id-properties-wrapper"]', { hasText: labelText });
   public getVisibilityField = (labelText: string) => this.getChild('[data-automation-id="library-visibility-properties-wrapper"]', { hasText: labelText });
   public getDescriptionField = this.getChild('[data-automation-id="library-description-properties-wrapper"] textarea');
+  public propertiesTab = this.getChild('.mat-tab-label-content').nth(0);
+  public commentsTab = this.getChild('.mat-tab-label-content').nth(1);
+  public commentInputField = this.getChild('mat-form-field');
+  public commentsHeader = this.getChild('#comment-header');
+  public addCommentButton = this.getChild('[data-automation-id="comments-input-add"]');
+  public commentsList = this.getChild('.adf-comment-list-item');
+  public commentUsername = this.getChild('.mat-line.adf-comment-user-name');
+  public commentTextContent = this.getChild('.mat-line.adf-comment-message');
+  public commentTimestamp = this.getChild('.mat-line.adf-comment-message-time');
+  public commentProfileIcon = this.getChild('.adf-comment-user-icon.ng-star-inserted');
+
+
+  async checkCommentsHeaderCount() {
+    let commentsCountTextContent: string = await this.commentsHeader.textContent();
+    let commentsCountSplit: string = commentsCountTextContent.split("Comments (")[1];
+    let commentsCountString: string = commentsCountSplit.split(")")[0];
+    let commentsCount: number = parseInt(commentsCountString);
+    return commentsCount;
```

Comment:
You can return a result of `parseInt` right away without creating a new variable

Reply:
Good idea, will be fixed in the next push